en/docs/install-and-setup/setup/api-controller/managing-apis-api-products/managing-apis-and-api-products.md

```diff
@@ -510,4 +510,123 @@ For more information, see [Download and Initialize the apictl]({{base_path}}/ins
     !!! Info
         - If ```--gateway-env``` or ```-g``` flag not provided, revision will be undeployed from all the gateway environments it is already deployed.
         - If there are multiple gateway environments, provide them one by one by specifying the flag ```--gateway-env``` or ```-g```. If a label has more than one words,
-        wrap the entire label name with quotes.
\ No newline at end of file
+        wrap the entire label name with quotes.
+
+## Get the logging details of APIs or an API in an environment
+
+Follow the instructions below to get the logging details of APIs or an API in an environment using apictl:
+
+1.  Make sure that the WSO2 API-M 4.0.0 version is started and that the 4.0.0 version of apictl is set up.   
+For more information, see [Download and Initialize the apictl]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-apictl).
+2.  Log in to the WSO2 API-M in the environment by following the instructions in [Login to an Environment]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#login-to-an-environment).
+3.  Run the corresponding apictl command below to get the logging details of APIs or an API in an environment.
+
+    1. Get the logging details of APIs or an API in an environment.
+
+        -   **Command**
+            ``` bash
+            apictl get api-logging -e <environment>
+            ```
+            ``` bash
+            apictl get api-logging -e <environment> --tenant-domain <tenant domain> 
+            ```
+            ``` bash
+            apictl get api-logging --environment <environment> --tenant-domain <tenant domain> 
+            ```
+            ``` bash
+            apictl get api-logging  -e <environment> --api-id <api-id> 
+            ```
+            ``` bash
+            apictl get api-logging  -e <environment> --api-id <api-id> --tenant-domain <tenant domain> 
+            ```
+
+            !!! info
+                **Flags:**  
+                
+                -   Required :  
+                    `--environment` or `-e` : The environment that the command is executed on  
+                -   Optional :  
+                    `--tenant-domain` : The tenant domain which you need to get the logging details of APIs or an API. If the the flag is not provided, default carbon.super value will be used when executing the command    
```

Comment:
This line can be changed to:

... If the the flag is not provided, **carbon.super will be used as the default value** when executing the command